index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="header">
+        <a href="#Home" class="active"> My Logo</a>
+        <a href=" " class="icon">
+            <i class="fa-solid fa-bars fa-2x"></i>
+        </a>
+    </div>
+    <div class="headline-section">
+        <h1 class="txt"> I'm Dev Mike <br>
+            Glad to see<br>you!</h1>
+        <p class="introduction"> I'm a software developer I can help you build a project a product,feature or
+            website Look through some
+            of my work and <br> experience!If you like what you see and have a project you need coded don't hestiate
+            to contact me</p>
+        <div class="socials">
+            <h2>LET'S CONNECT</h2>
+            <div class="icons">
+                <a href=""> <i class="fa-brands fa-twitter fa-1x" background: #505F79></i></a>
+                <a href=""> <i class="fa-brands  fa-linkedin fa-1x "></i></a>
+                <a href=""> <i class="fa-brands fa-github fa-1x"></i></a>
+                <a href=""> <i class="fa-brands fa-m fa-1x"></i></a>
+                <a href=""><i class="fa-solid fa-hand-peace fa-1x"></i></a>
+            </div>
+        </div>
```

Comment:
- I suggest you use a `ul` tag to wrap all your social icon elements and wrap each  `a` tag in a `li` tag. Please fix this as it would help improve your site's HTML structure 👍 👍